src/index.html

```diff
@@ -12,7 +12,10 @@
   </head>
   <body>
 
-    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+    <form
+
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post">
```

Comment:
why an empty line?
closing bracket should be aligned with opening
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
    >
```